src/components/ParentLink.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from 'react';
+import {PersonLink} from "./PersonLink";
+import {Person} from "../types";
+
+interface Props {
+  parentName: string | null,
+  parent: Person | undefined,
+}
+
+export const ParentLink: FC<Props> = ({ parentName, parent }) => {
+  if (!parentName) {
+    return <span>-</span>;
+  }
+
+  if (parent === undefined) {
+    return <span>{parentName}</span>;
+  }
+
+  return <PersonLink person={parent} />;
+};
```

Comment:
This way/order is more often used
```suggestion
  if (parent) {
    return <PersonLink person={parent} />;
  }

  return <span>{parentName}</span>;
};
```